boards/xtensa/nxp_adsp_imx8ulp/nxp_adsp_imx8ulp_defconfig

```diff
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NXP_IMX8ULP=y
+CONFIG_SOC_NXP_IMX8ULP=y
+CONFIG_BOARD_NXP_ADSP_IMX8ULP=y
+
+CONFIG_GEN_ISR_TABLES=y
+CONFIG_GEN_IRQ_VECTOR_TABLE=n
+
+CONFIG_XTENSA_RESET_VECTOR=y
+
+CONFIG_XTENSA_USE_CORE_CRT1=y
+
+CONFIG_XTENSA_SMALL_VECTOR_TABLE_ENTRY=y
```

Comment:
This could be implicitly selected by `CONFIG_SOC_SERIEX_NXP_IMX8ULP` unless this is board specific.